dookug-api/dookug-api-jee10/dookug-api-rest-jee10/src/main/java/hu/icellmobilsoft/dookug/api/rest/document/IDocumentGenerateInlineInternalRest.java

```diff
@@ -0,0 +1,144 @@
+     */
+    int LOG_ENTITY_SIZE = 1000;
+
+    /**
+     * REST interface definition for document generation by multipart form input
+     * 
+     * @param form
+     *            multipart form input
+     * @return Generated document content output stream
+     * @throws BaseException
+     *             on error
+     */
+    @Operation(summary = "Dokumentum generálása multipart request-ben megadott értékek alapján",
+            description = "A POST " + DocumentGeneratePath.INTERNAL_DOCUMENT_GENERATE_INLINE
+                    + " végponthoz hasonló működésű végpont, azonban a request-et multipart formában fogadjuk.")
+    @POST
+    @Consumes(MediaType.MULTIPART_FORM_DATA)
+    @Produces(MediaType.APPLICATION_OCTET_STREAM)
+    @Path(DocumentGeneratePath.MULTIPART)
+    @LogSpecifiers({ @LogSpecifier(target = LogSpecifierTarget.REQUEST, maxEntityLogSize = LOG_ENTITY_SIZE),
+            @LogSpecifier(target = LogSpecifierTarget.CLIENT_REQUEST, maxEntityLogSize = LOG_ENTITY_SIZE),
+            @LogSpecifier(target = LogSpecifierTarget.RESPONSE, maxEntityLogSize = LogSpecifier.NO_LOG),
+            @LogSpecifier(target = LogSpecifierTarget.CLIENT_RESPONSE, maxEntityLogSize = LogSpecifier.NO_LOG) })
+    Response postDocumentGenerateMultipart(@MultipartForm DocumentGenerateMultipartForm form) throws BaseException;
+
+    /**
+     * REST interface definition for document generation by structured input
+     * 
+     * @param request
+     *            structured input
+     * @return Generated document content output stream
+     * @throws BaseException
+     *             on error
+     */
+    @Operation(summary = "Dokumentum generálása request-ben megadott értékek alapján",
+            description = "Inline generálás esetén a template-et, a paramétereket és a beállításokat is fogadjuk a végponton, majd ezek alapján generáljuk a dokumentumot.\n\n"
+                    + "Több, hierarchikusan rendezett template feldolgozására is van lehetőség, amely HTML alapú dokumentumgenerálás esetén hasznosítható.\n\n"
+                    + "A template-et base64binary formátumban fogadjuk. A paramétereket kulcs-érték párosokként, vagy JSON struktúrában base64binary formátumban fogadjuk.")
+    @POST
+    @Consumes(value = { MediaType.APPLICATION_XML, MediaType.TEXT_XML, MediaType.APPLICATION_JSON })
+    @Produces(value = { MediaType.APPLICATION_OCTET_STREAM })
+    @LogSpecifiers({ @LogSpecifier(target = LogSpecifierTarget.REQUEST, maxEntityLogSize = LOG_ENTITY_SIZE),
+            @LogSpecifier(target = LogSpecifierTarget.CLIENT_REQUEST, maxEntityLogSize = LOG_ENTITY_SIZE),
+            @LogSpecifier(target = LogSpecifierTarget.RESPONSE, maxEntityLogSize = LogSpecifier.NO_LOG),
+            @LogSpecifier(target = LogSpecifierTarget.CLIENT_RESPONSE, maxEntityLogSize = LogSpecifier.NO_LOG) })
+    Response postDocumentGenerateEntityBody(@ValidateXML(xsdPath = XsdConstants.SUPER_XSD_PATH) DocumentGenerateWithTemplatesRequest request)
```

Comment:
SupeSuxsd-vel ellenőrizni az inputot kényelmes, viszont sokkal lasabb, mert mindent felolvas.

Reply:
de csak elsőre, amig nem cacheli be az xsd-ket